src/ProxyManager/ProxyGenerator/Util/Properties.php

```diff
@@ -45,10 +45,11 @@ private function __construct(array $properties)
 
     /**
      * @param ReflectionClass $reflection
+     * @param array           $excludedProperties
      *
      * @return self
      */
-    public static function fromReflectionClass(ReflectionClass $reflection)
+    public static function fromReflectionClass(ReflectionClass $reflection, array $excludedProperties = [])
```

Comment:
Can you add a new method that `->filterProperties()` and returns a new `Properties` instead?